infrastructure/src/main/java/mops/database/dto/DOrganisiertVDto.java

```diff
@@ -0,0 +1,12 @@
+package mops.database.dto;
+
+import org.springframework.data.relational.core.mapping.Table;
+
+@Table("dozentOrganisiertVeranstaltung")
+public class DOrganisiertVDto {
+  transient Long dozent;
+
+  public DOrganisiertVDto(Long dozent) {
+    this.dozent = dozent;
+  }
```

Comment:
Why didn't you use an AllArgsConstructor here?

Reply:
could be used instead, i will add the changes in the next commit